.github/actions/secure-upload-artifact/action.yml

```diff
@@ -21,7 +21,7 @@ runs:
     # `.github/actions/secure-builder-checkout`.
     - name: Compute binary hash
       id: compute-digest
-      uses: ./__BUILDER_CHECKOUT_DIR__/.github/actions/compute-sha256
+      uses: slsa-framework/slsa-github-generator/.github/actions/compute-sha256@main
```

Comment:
is this a temporary change? i thought it was intended for the top-level workflows to check out this repo before calling this action

Reply:
I ran into problems due to all the directory moves we do: the post-clean up the runner was doing kept failing, so I made this change. Will be a long-term change. I will update the pre-submits accordingly.

Wdut?

Reply:
Makes sense to me.